### DISCUSSION TITLE
Is it possible to get blob or convert to blob?

### DISCUSSION THREAD

**Discussion by aposuarez** [react-native-image-picker/react-native-image-picker#1803] [Q&A]
Hi,

I need to upload an image into a storage that only accepts blob.
I'm currently able to get base64, fileName, fileSize, height, width, type, uri from launchImagePickerLibrary. 
I just want to ask if it is possible to get a blob or maybe convert the base64 string into blob to that I can upload the image into the blob storage?

I'm new in react native so thanks in advance anyone who can give me an idea on how to solve this :)

**Comment by juleshwar** [comment]
You can take the `uri` from the `Response` object. Then use `await (await fetch(responseObject.uri)).blob()` to get the Blob
